import_csv.py

```diff
@@ -15,179 +15,135 @@
+    site.applicant_home_phone = clean_get(s, 
       "Repair Application: Applicant Home Phone")
-    site.applicant_work_phone = clean_s(
+    site.applicant_work_phone = clean_get(s, 
       "Repair Application: Applicant Work Phone")
-    site.applicant_mobile_phone = clean_s(
+    site.applicant_mobile_phone = clean_get(s, 
       "Repair Application: Applicant Mobile Phone")
-    site.sponsor = clean_s("(Sponsor) Campaign Description")
-    site.rrp_test = clean_s("Repair Application: RRP Test Results")
-    site.rrp_level = clean_s("Repair Application: RRP Result Notes")
-    # site.roof = clean_s("Roof?")
-    site.jurisdiction = clean_s("Jurisdiction")
-    site.announcement_subject = clean_s("Announcement Subject")
-    site.announcement_body = clean_s("Announcement Body")
+    site.sponsor = clean_get(s, "Sponsor")
+    site.rrp_test = clean_get(s, "Repair Application: RRP Test Results")
+    site.rrp_level = clean_get(s, "Repair Application: RRP Test Results")
+    # site.roof = clean_get(s, "Roof?")
+    site.jurisdiction = clean_get(s, "Jurisdiction")
+    site.announcement_subject = clean_get(s, "Announcement Subject")
+    site.announcement_body = clean_get(s, "Announcement Body")
     site.put()
     logging.info('put site %s', number)
 
 
-def import_captains(input_csv="../2012_ROOMS_Captain_email_sample.csv"):
-  """Change input_csv to actual input file - the default is test data."""
+def import_captains(input_csv):
+  """
+  input_csv is a path like "../2012_ROOMS_site_info_sample.csv"
+
+  Change input_csv to actual input file - the default is test data.
+  """
   reader = csv.DictReader(open(input_csv))
   for s in reader:
-    def clean_s(k):
-      return s[k].replace('\n', ' ').replace('\xe2', "'").replace('\x80', "'").replace('\x99', '').replace('\xc3',
-                                                                                                           '').replace(
-        '\x95', '').encode('ascii', 'replace')
+    
 
     key = s.get('key')
-    email = clean_s("Email")
-    rooms_id = clean_s("ROOMS Captain ID")
-    # name = "%s %s" % (clean_s("First Name"),
-    #                   clean_s("Last Name"))
-    name = clean_s("Name")
+    email = clean_get(s, "Email")
+    rooms_id = clean_get(s, "ROOMS Captain ID")
+    name = clean_get(s, "Name")
     captain = None
     if key:
-      captain = models.Captain.get_by_id(int(key))
+      captain = ndb_models.Captain.get_by_id(int(key))
       if captain:
         logging.info('got captain from key %s', key)
     if not captain:
-      captain = models.Captain.all().filter('rooms_id =', rooms_id).get()
+      captain = ndb_models.Captain.query(ndb_models.Captain.rooms_id == rooms_id).get()
       if captain:
         logging.info('got captain from rooms_id %s', rooms_id)
     if not captain:
-      captain = models.Captain.all().filter('email =', email).get()
+      captain = ndb_models.Captain.query().filter(ndb_models.Captain.email == email).get()
```

Comment:
Here and in other places, can put  the filter param directly into query(). 

Like 
ndb_models.Captain.query(ndb_models.Captain.email == email)

see https://cloud.google.com/appengine/docs/standard/python/ndb/queries